src/screens/Button.js

```diff
@@ -679,7 +679,21 @@ const ButtonPage = () => (
             Adjustments to the default Button style when the Button is active.
           </Description>
           <PropertyValue type="object">
-            <Example>{`{}`}</Example>
+            <Example>{`{ default: {
```

Comment:
We should remove the `{ default: ` outer object here

Reply:
same for the rest of them

Reply:
just updated this